Board/CellColorEditor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Chess.Board
+{
+    class CellColorEditor
+    {
+        public CellColor ColorizeTheCell(int ColumnIndex, int RowIndex)
```

Comment:
Sugeruję zmianę nazwy

```suggestion
        public CellColor PaintTheCell(int ColumnIndex, int RowIndex)
```